interfaces/cython/cantera/reaction.pyx

```diff
@@ -277,7 +277,7 @@ cdef class TwoTempPlasmaRate(ArrheniusTypeRate):
 
     .. math::
 
-        k_f = A T_e^b \exp(-\tfrac{E_{a,g}}{RT}) \exp(-\tfrac{E_{a,e}}{RT_e})
+        k_f = A T_e^b \exp(-\tfrac{E_{a,g}}{RT}) \exp(\tfrac{E_{a,e}(T_e - T)}{R T T_e})
 
     where :math:`A` is the `pre_exponential_factor`, :math:`b` is the
```

Comment:
As I mentioned to @speth on #1196, it'd be nice to have the formatting of `A`, `b`, etc. be consistent for all the reaction types.